.github/workflows/dev-workflow.yaml

```
@@ -191,58 +192,58 @@ jobs:
-  #         ENABLE_CRON: true
-  #         SCHEDULE: 'cron(0 15 * * ? *)'
-  #         DOMAIN: voice-korea-watcher-api.dev.biyard.co
-
-  #       run: |
-  #         npm i -g aws-cdk
-  #         make deploy-if-needed
+  watcher-deployment:
+    runs-on: ubuntu-22.04
+
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)